src/gluonts/mx/trainer/callback.py

```diff
@@ -338,3 +339,31 @@ def on_network_initializing_end(
         self, training_network: nn.HybridBlock
     ) -> None:
         copy_parameters(self.predictor.prediction_net, training_network)
+
+
+class TimeLimitCallback(Callback):
```

Comment:
Can we have a short doc-string describing the class and it's parameters?

Reply:
Let's give it a more descriptive name.

Reply:
How about `TrainingTimeLimitCallback`?